library/src/main/java/io/appwrite/services/Avatars.kt

```diff
@@ -0,0 +1,242 @@
+package io.appwrite.services
+
+import android.net.Uri
+import io.appwrite.Client
+import io.appwrite.enums.OrderType
+import io.appwrite.exceptions.AppwriteException
+import okhttp3.Cookie
+import okhttp3.Response
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import java.io.File
+
+class Avatars(private val client: Client) : BaseService(client) {
+
+    /**
+     * Get Browser Icon
+     *
+     * You can use this endpoint to show different browser icons to your users.
+     * The code argument receives the browser code as it appears in your user
+     * /account/sessions endpoint. Use width, height and quality arguments to
+     * change the output settings.
+     *
+     * @param code
+     * @param width
+     * @param height
+     * @param quality
+     * @return The request response with a JSON body 
```

Comment:
We don't receive a JSON response on this endpoint. @TorstenDittmann just made a similar fix on the Node SDK.

Reply:
Done :+1: